README.md

```diff
@@ -25,43 +25,51 @@ gem 'safe-pg-migrations'
 Consider the following migration:
 
 ```rb
-class AddAdminToUsers < ActiveRecord::Migration[5.2]
+class AddPatientRefToAppointments < ActiveRecord::Migration[6.0]
   def change
-    add_column :users, :admin, :boolean, default: false, null: false
+    add_reference :appointments, :patient
   end
 end
 ```
 
-If the `users` table is large, running this migration on a live Postgres 9 database will likely cause downtime. **Safe PG Migrations** hooks into Active Record so that the following gets executed instead:
+If the `users` table is large, running this migration will likely cause downtime. **Safe PG Migrations** hooks into Active Record so that the following gets executed instead:
 
 ```rb
-class AddAdminToUsers < ActiveRecord::Migration[5.2]
+class AddPatientRefToAppointments < ActiveRecord::Migration[6.0]
   # Do not wrap the migration in a transaction so that locks are held for a shorter time.
   disable_ddl_transaction!
 
   def change
     # Lower Postgres' lock timeout to avoid statement queueing. Acts like a seatbelt.
-    execute "SET lock_timeout TO '5s'" # The lock_timeout duration is customizable.
+    old_lock_value = query_value('SHOW lock_timeout')
```

Comment:
I guess it's ok to skip this mechanism (querying and setting back initial values) in the documentation, to me it's implementation details that do not help understand the purpose of the gem.

Reply:
Done. Though, I've added a comment at the end of the method. If someone copy-paste the method it could be dangerous to let the timeout set to 0